shardingsphere-infra/shardingsphere-infra-parser/src/main/java/org/apache/shardingsphere/infra/parser/sql/SQLStatementParserExecutor.java

```diff
@@ -32,7 +32,7 @@
     
     public SQLStatementParserExecutor(final String databaseType) {
         parserEngine = new SQLParserEngine(databaseType);
-        visitorEngine = new SQLVisitorEngine(databaseType, "STATEMENT");
+        visitorEngine = new SQLVisitorEngine(databaseType, "STATEMENT", null);
```

Comment:
Maybe `new Properties()` is better.

Reply:
@tristaZero I think it is more user-friendly to use `null`,It is difficult to accept a new empty configuration without configuration.